### ISSUE TITLE
Cannot add markdown-text on macOS

### ISSUE THREAD

**Issue by tuan-hda** [Yonom/assistant-ui#429]
Hi,

I'm trying to install the markdown-text component.
It throws this error:
```
SyntaxError: Unexpected token '<', "<!DOCTYPE "... is not valid JSON
    at JSON.parse (<anonymous>)
    at Response.json (file:///Users/tuanhoangdinhanh/.npm/_npx/2c1ae577c581653c/node_modules/node-fetch/src/body.js:149:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file:///Users/tuanhoangdinhanh/.npm/_npx/2c1ae577c581653c/node_modules/shadcn-ui/dist/index.js:2:4471
    at async Promise.all (index 0)
    at async _ (file:///Users/tuanhoangdinhanh/.npm/_npx/2c1ae577c581653c/node_modules/shadcn-ui/dist/index.js:2:4438)
    at async N (file:///Users/tuanhoangdinhanh/.npm/_npx/2c1ae577c581653c/node_modules/shadcn-ui/dist/index.js:2:3238)
    at async Command.<anonymous> (file:///Users/tuanhoangdinhanh/.npm/_npx/2c1ae577c581653c/node_modules/shadcn-ui/dist/index.js:3:1499)
Failed to fetch components from registry.
other-package-script process exited with code 1
```

Can you have a look at this? Im currently using macOS and assistant-ui version 0.3.5
Thanks!

**Comment by Yonom** [comment]
Thanks a lot for reporting this, some configuration was not migrated to the new docs website yesterday. 
This should be fixed now, sorry about that!

(Fixed via #430 and #431)

**Comment by Yonom** [comment]
This was a server side issue, so no need to update anything on your side, just rerun the command
